### PR TITLE
Update devDependencies and gems.

### PR DESCRIPTION
**Pull request by XhmikosR**
TODO:

- [x] Update rollup and rollup-plugin-babel

### CODE REVIEW

**Review by Johann-S — APPROVED** [review]

**Comment by XhmikosR** [comment]
Do not merge this, I will merge when more things are added.

**Comment by XhmikosR** [comment]
@Johann-S: LGTY?

**Review by Johann-S — APPROVED** [review]
LGTM 👍

**Pull request merged**
